sql/mob_pools.sql

```diff
@@ -985,7 +985,7 @@ INSERT INTO `mob_pools` VALUES (924,'Davoi_Hornet','Davoi_Hornet',48,0x000010010
 INSERT INTO `mob_pools` VALUES (925,'Davoi_Mush','Davoi_Mush',229,0x0000240100000000000000000000000000000000,1,1,7,240,100,0,1,0,0,0,0,0,27,131,0,0,0,1,0,229);
 INSERT INTO `mob_pools` VALUES (926,'Davoi_Pugil','Davoi_Pugil',197,0x00005C0100000000000000000000000000000000,1,1,7,240,100,0,1,0,0,0,0,0,0,3,0,0,0,1,0,197);
 INSERT INTO `mob_pools` VALUES (927,'Davoi_Wasp','Davoi_Wasp',48,0x0000100100000000000000000000000000000000,1,1,2,240,100,0,1,0,1,0,0,0,238,643,0,0,0,1,0,48);
-INSERT INTO `mob_pools` VALUES (928,'Da_Dha_Hundredmask','Da_Dha_Hundredmask',202,0x00008C0200000000000000000000000000000000,6,6,2,265,100,0,1,0,1,2,0,0,0,0,0,0,0,1,0,202);
+INSERT INTO `mob_pools` VALUES (928,'Da_Dha_Hundredmask','Da_Dha_Hundredmask',202,0x00008C0200000000000000000000000000000000,6,6,2,265,100,0,1,0,1,2,2,0,0,0,0,0,0,1,0,202);
```

Comment:
`MOD_GRAVITYRES        = 249` is resistance to gravity.

We also have:
```lua
MOD_SLEEPRES          = 240
MOD_POISONRES         = 241
MOD_PARALYZERES       = 242
MOD_BLINDRES          = 243
MOD_SILENCERES        = 244
MOD_VIRUSRES          = 245
MOD_PETRIFYRES        = 246
MOD_BINDRES           = 247
MOD_CURSERES          = 248
MOD_GRAVITYRES        = 249
MOD_SLOWRES           = 250
MOD_STUNRES           = 251
MOD_CHARMRES          = 252
MOD_AMNESIARES        = 253
MOD_LULLABYRES        = 254
```
for future reference 😄 

these can be set in script when in onMobInitialize, or in one of the sql tablles that deals with mods.